### DISCUSSION TITLE
When using multiple data-importer configs - how can I know which one that gives error message in the email reports?

### DISCUSSION THREAD

**Discussion by nimonimonimo** [firefly-iii/firefly-iii#9269] [Firefly III]
Hi,

I have a cron job importing me and my wife's all accounts from two different banks. I run it like this:
`IMPORT_DIR_ALLOWLIST=/usr/local/www/data-importer/storage/configurations /usr/local/bin/php /usr/local/www/data-importer/artisan importer:auto-import /usr/local/www/data-importer/storage/configurations` 

In /usr/local/www/data-importer/storage/configurations I have 4 different config files, one for each person at each bank. Sometimes there are an error message that it wasn't possible to import because of expiration of "Your Nordigen End User Agreement has expired" or similar.

Is there any way I can see which config file it is that triggers this error message?

Thanks in advance!

**Comment by JC5** [comment]
There is a ticket for this, to clarify from which source you're importing :)

**Reply by nimonimonimo** [reply]
Ah, thanks!

Found it: https://github.com/firefly-iii/firefly-iii/issues/9045
